config/database.yml

```diff
@@ -30,8 +30,8 @@ development:
    encoding: unicode
    database: insta-social_development
    pool: 5
-   username: redvanisation
-   password:
+   username: mouahamdou
+   password: 'password'
```

Comment:
[OPTIONAL]

Instead of changing `username` every time the other person starts working on the project, wouldn't it be okay to leave it with the default `pg` username? :thinking:  

This works for me. :point_down: 

```yml
default: &default


  adapter: postgresql
  encoding: unicode
  # For details on connection pooling, see Rails configuration guide
  # http://guides.rubyonrails.org/configuring.html#database-pooling
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>

development:
   <<: *default
   database: insta-social_development
```